TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -193,125 +116,102 @@ public class OptionsManager
-
-                OptionsMassEditTab.RoundAboutQuickFix_KeepClearYieldR.Save(),
-                OptionsMassEditTab.RoundAboutQuickFix_RealisticSpeedLimits.Save(),
-                OptionsMassEditTab.RoundAboutQuickFix_ParkingBanMainR.Save(),
-                OptionsMassEditTab.RoundAboutQuickFix_ParkingBanYieldR.Save(),
-
-                OptionsVehicleRestrictionsTab.NoDoubleCrossings.Save(),
-                OptionsVehicleRestrictionsTab.DedicatedTurningLanes.Save(),
-            };
+            var save = new byte[57];
+
+            save[0] = ConvertFromSimulationAccuracy(Options.simulationAccuracy);
+            save[1] = 0; // Options.laneChangingRandomization
+            save[2] = (byte)Options.recklessDrivers;
+            save[3] = (byte)(Options.relaxedBusses ? 1 : 0);
+            save[4] = (byte)(Options.nodesOverlay ? 1 : 0);
+            save[5] = (byte)(Options.allowEnterBlockedJunctions ? 1 : 0);
+            save[6] = (byte)(Options.advancedAI ? 1 : 0);
+            save[7] = (byte)(Options.highwayRules ? 1 : 0);
+            save[8] = (byte)(Options.prioritySignsOverlay ? 1 : 0);
+            save[9] = (byte)(Options.timedLightsOverlay ? 1 : 0);
+            save[10] = (byte)(Options.speedLimitsOverlay ? 1 : 0);
+            save[11] = (byte)(Options.vehicleRestrictionsOverlay ? 1 : 0);
+            save[12] = (byte)(Options.strongerRoadConditionEffects ? 1 : 0);
+            save[13] = (byte)(Options.allowUTurns ? 1 : 0);
+            save[14] = (byte)(Options.allowLaneChangesWhileGoingStraight ? 1 : 0);
+            save[15] = (byte)(Options.disableDespawning ? 0 : 1);
+            save[16] = 0; // Options.IsDynamicPathRecalculationActive
+            save[17] = (byte)(Options.connectedLanesOverlay ? 1 : 0);
+            save[18] = (byte)(Options.prioritySignsEnabled ? 1 : 0);
+            save[19] = (byte)(Options.timedLightsEnabled ? 1 : 0);
+            save[20] = (byte)(Options.customSpeedLimitsEnabled ? 1 : 0);
+            save[21] = (byte)(Options.vehicleRestrictionsEnabled ? 1 : 0);
+            save[22] = (byte)(Options.laneConnectorEnabled ? 1 : 0);
+            save[23] = (byte)(Options.junctionRestrictionsOverlay ? 1 : 0);
+            save[24] = (byte)(Options.junctionRestrictionsEnabled ? 1 : 0);
+            save[25] = (byte)(Options.parkingAI ? 1 : 0);
+            save[26] = (byte)(Options.preferOuterLane ? 1 : 0);
+            save[27] = (byte)(Options.individualDrivingStyle ? 1 : 0);
+            save[28] = (byte)(Options.evacBussesMayIgnoreRules ? 1 : 0);
+            save[29] = (byte)(Options.instantEffects ? 1 : 0);
+            save[30] = (byte)(Options.parkingRestrictionsEnabled ? 1 : 0);
+            save[31] = (byte)(Options.parkingRestrictionsOverlay ? 1 : 0);
+            save[32] = (byte)(Options.banRegularTrafficOnBusLanes ? 1 : 0);
+            save[33] = (byte)(Options.showPathFindStats ? 1 : 0);
+            save[34] = Options.altLaneSelectionRatio;
```

Comment:
Loads DLSPercentage, and stores altLaneSelectionRatio, hope they are the same variable

Reply:
yup 
![image](https://user-images.githubusercontent.com/19638970/151645998-7e64ce16-a458-4496-9830-d73ab943934b.png)